packages/block-library/src/paragraph/edit.js

```diff
@@ -66,7 +66,7 @@ function useDropCapMinimumHeight( isDropCap, deps ) {
 					getComputedStyle(
 						element,
 						'first-letter'
-					).height
+					).lineHeight
```

Comment:
Getting the computed style of the `first-letter` pseudo element doesn't seem to work in Firefox. Instead it calculates the height of the paragraph, which would be `0` if there's only a floated drop cap element. Using the line height fixes the problem since it's never `0`. It also works nicely when it does work for the pseudo element. Tested in Safari, Chrome, and Firefox.